src/App.js

```diff
@@ -1,10 +1,96 @@
 import React from 'react';
 import './App.css';
+import Button from "./components/Button";
+import Product from "./components/Pruduct";
+import Product2 from "./components/Product2";
+import bag1 from './assets/bag_1.png';
+import bag2 from './assets/bag_2.png';
+import bag3 from './assets/bag_3.png';
+import bag4 from './assets/bag_4.png';
+import brand from './assets/brand.png';
+import ourStory from './assets/our_story.png';
+import Tile from "./components/Tile";
 
+/*
+// STAPPENPLAN OPDRACHT 1
+// 1. nav-element maken
+// 2. Wikkel de titel en nav in een fragment!
+// 3. Schrijf één button element:
+  // [ ] Tekst
+  // [ ] Type (button)
+  // [ ] Hoe krijgen we een event-listener op een react button? Om daarna console.log() iets in de console
+  // [ ] Hoe krijgen we een button disabled?
+// Stappenplan opdracht 2
+// Volg de aanpak uit het demo project (in de README.md)
+
+# Aanpak componenten maken
+1. `components`-map maken
+2. Bestandje maken met de naam van jouw component (`Testimonial.js`)
+3. Functie schrijven (met hoofdletter) die een HTML-element returned
+4. Exporteer de functie (component) uit het bestand
+5. Importeer de functie (component) in App.js en gebruik hem als: `<Component />`
+6. Plak je de orignele HTML in jouw custom component. Welke data heeft jouw component nodig?
+7. Bedenk property-namen en geef informatie door aan het component vanaf `App.js`
+8. Ontvang het props-object als parameter
+9. Vervang de statische tekst door dynamische waardes uit het props object
+
+<button disabled={true} >
+ */
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button title="to the collection" disabled={false} clickBehaviour="one"/>
```

Comment:
Goed dat je het button component niet Nav hebt genoemd.
Uit haast had ik dit parongeluk wel gedaan terwijl t nog in de opdracht stond aangegeven.